application/server.py

```diff
@@ -86,13 +99,18 @@ def handle_join_room_form():
         room_name = request.form.get('room-name')
         room_password = request.form.get('room-password')
 
-        # TODO: Look for room in database
+        game = models.Game.query.filter_by(room_name=room_name, room_password=room_password).first()
+        newp = models.Player(name=name, color="#CCFF00", game_id=game.id)
+        db.session.add(newp)
+        db.session.commit()
+                    
+        players = models.Player.query.all()
+        print(players)
 
-        if room_password == "correct": # TODO: Verify password is correct
```

Comment:
Instead of deleting this if statement, you want to check that room_password == game.room_password. If so, redirect to waiting room. Else, redirect to enter room.